mqtt/mqtt-bridge/src/controller/mod.rs

```diff
@@ -153,6 +142,31 @@ async fn process_update(update: BridgeControllerUpdate, bridges: &mut Bridges) {
     }
 }
 
+async fn start_bridge(
+    settings: &BridgeSettings,
+    system_address: &str,
+    device_id: &str,
+    bridges: &mut Bridges,
+) {
+    if let Some(upstream_settings) = settings.upstream() {
+        let bridge_result = bridge_builder::<WakingMemoryStore>()
+            .with_system_address(String::from(system_address))
+            .with_device_id(String::from(device_id))
+            .with_connection_settings(upstream_settings.clone())
+            .build();
```

Comment:
I have made some builders for memory and rb, only memory is here until rb pr goes in. Let me know if there is a preference for doing something else here :)